packages/lib/modules/pool/PoolDetail/PoolBanners/PoolFeeManagerBanner.tsx

```diff
@@ -0,0 +1,16 @@
+import { useFeeManager } from '@repo/lib/modules/fee-managers/useFeeManager'
+import { usePool } from '../../PoolProvider'
+import { EzklPromoBanner } from '@repo/lib/shared/components/promos/EzklPromoBanner'
+
+export function PoolFeeManagerBanner() {
+  const { pool } = usePool()
+  const { hasFeeManager, feeManager } = useFeeManager(pool)
+
+  if (!hasFeeManager) return null
+
+  if (feeManager?.id === 'fee_manager_ezkl') {
```

Comment:
Maybe using a constant here that could be shared in other parts of the code could be better?

Reply:
The thought crossed my mind but i think this will be the only place it will be used.
Can always revisit if it's needed anywhere else.